eslint.config.mjs

```diff
@@ -0,0 +1,90 @@
+import tsStylistic from '@stylistic/eslint-plugin-ts';
+import prettier from 'eslint-config-prettier';
+import tsEslint from 'typescript-eslint';
+
+import apify from '@apify/eslint-config/ts';
+
+export default [
+	{
+		ignores: [
+			'**/dist',
+			'node_modules',
+			'coverage',
+			'website/{build,.docusaurus}',
+			'**/*.d.ts',
+			'test/tmp/**/*',
+			'.yarn/**/*',
+		],
+	},
+	...apify,
+	prettier,
+	{
+		languageOptions: {
+			parser: tsEslint.parser,
+			parserOptions: {
+				project: 'tsconfig.eslint.json',
+			},
+		},
+	},
+	{
+		plugins: {
+			'@typescript-eslint': tsEslint.plugin,
+			'@stylistic': tsStylistic,
+		},
+		rules: {
+			'@typescript-eslint/no-empty-object-type': 'off',
+			'@typescript-eslint/no-explicit-any': 'off',
+			'max-classes-per-file': 'off',
+			'no-empty-function': 'off',
+			'import/order': 'off', // TODO
+			'no-use-before-define': 'off', // TODO
+			'no-param-reassign': 'off',
+			'no-void': 'off',
+			'no-underscore-dangle': 'off',
+			'no-console': 'off',
+			'import/no-extraneous-dependencies': 'off',
+			'import/extensions': 'off',
+			'import/no-default-export': 'off',
```

Comment:
we use default exports here? lets not disable the rule just because of some configs

Reply:
We have several parts rn that export default (related to oclif stuff.) I'll add a todo to remove it on yargs branch